src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -33,6 +33,11 @@
     <NuGetConfigFile Condition="'$(OriginalNuGetConfigFile)' != ''">$(BaseIntermediateOutputPath)$([System.IO.Path]::GetFileName('$(OriginalNuGetConfigFile)'))</NuGetConfigFile>
 
     <SourceBuiltSdksDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'source-built-sdks'))</SourceBuiltSdksDir>
+    <RepoArtifactsShippingPackagesDir Condition="'$(RepositoryName)' != 'source-build-reference-packages'">$([MSBuild]::NormalizeDirectory('$(ArtifactsShippingPackagesDir)', '$(RepositoryName)'))</RepoArtifactsShippingPackagesDir>
```

Comment:
I don't love that we have a pretty consistent pattern of comparing `'$(RepositoryName)' != 'source-build-reference-packages'` in various places throughout `Directory.Build.targets` and `Directory.Build.props`.

@jkoritzinsky added a new concept of a reference only repo here that would be good to use after his PR goes in. https://github.com/dotnet/installer/pull/18672/files#diff-0802dca0f4d533e4f44a76e021486aefa13bc1b193f018433b0d056ae84c9f3cR407-R408